Widgets/Textbox.py

```diff
@@ -22,6 +22,16 @@ def mousePressEvent(self, event):
         QTextEdit.mousePressEvent(self, event)
 
     def focusOutEvent(self, event):
+        editor = self.parentWidget().parentWidget()
+        self.setStyleSheet(TextBoxStyles.OUTFOCUS.value)
+        if self.toPlainText() == '':
+            for o in range(len(editor.object) - 1):
```

Comment:
idek know if it matters but isnt self a reference to the object so u could remove it with that instead of looping thru the objects. i feel like its not a huge deal but since it runs on all the focusout of the textbox it feels expensive. but mb im missing something

Reply:
oh prob so u can delete it from the notebook too, idk then